helpick-service/src/main/java/com/infoshare/database/FileDb.java

```diff
@@ -77,6 +78,42 @@ public void saveVolunteer(Volunteer volunteer) {
 
     }
 
+    @Override
+    public void saveVolunteerWithUuid(Volunteer volunteer){
```

Comment:
Ta metoda nie jest chyba potrzebna, 'saveVolunteer' robi to co trzeba. Po zmianie w Volunteer service na `db.saveVolunteer(volunteer.get());` zapis dziala poprawnie.

Reply:
'saveVolonteer' który mieliśmy początkowo działa na podstawie unikalnego maila, tutaj potrzebowałam rozszerzenia tej metody, ponieważ z tego co rozmawialiśmy na projekcie edycja wolontariusza miała pozwalać też na zmianę maila

Reply:
Ok masz racje. W takim wypadku bedziemy sie musieli zastanowic, czy nie usunac oryginalnej metody saveVolunteer, i zoastawic tylko Twoja implementacje. W koncu powinnismy miec jedna metode do zapisu Volunteer.